src/types/services/services.types.ts

```diff
@@ -37,8 +37,10 @@ export interface UserResponse {
 }
 
 export type Params = {
```

Comment:
params is not clear name, give name so we can see where those params are used

Reply:
created RequestParams general interface and extends from it